src/zui/ZUIMapControls.tsx

```diff
@@ -1,13 +1,13 @@
-import CircularProgress from '@mui/material/CircularProgress';
-import React, { useState } from 'react';
+import React, { useEffect, useRef, useState } from 'react';
 import { Box, Button, ButtonGroup, useTheme } from '@mui/material';
-import { Add, Remove, GpsFixed, Home } from '@mui/icons-material';
+import { Add, Remove, Home } from '@mui/icons-material';
 
 import { Latitude, Longitude, PointData } from 'features/areas/types';
+import { AnimatedGpsFixed } from 'zui/icons/AnimatedGpsFixed';
 
 type Props = {
   onFitBounds: () => void;
-  onGeolocate: (lngLat: PointData) => void;
+  onGeolocate: (lngLat: PointData, accuracy: number | null) => void;
```

Comment:
I think this property needs to be paired with an `autoUpdate` boolean flag, that can be set to false in all places where we don't want auto-updating, and perhaps even default to false.

We do want auto-updating for the canvasser UI, but right now the auto-updating is causing bugs for other maps, such as the map in the Geography section and the one in area assignments..

Reply:
This seems to be working now! Confirmed fixed in 511b1fc6d92011d3b443359c2bc9f0d91a16962d.